src/test/java/simpaths/data/statistics/EmploymentStatisticsTest.java

```diff
@@ -0,0 +1,94 @@
+                // 25% move from employment into unemployment
+                createTestPerson(1, Les_c4.EmployedOrSelfEmployed, Les_c4.NotEmployed),
+                createTestPerson(2, Les_c4.EmployedOrSelfEmployed, Les_c4.Student),
+                createTestPerson(3, Les_c4.EmployedOrSelfEmployed, Les_c4.EmployedOrSelfEmployed),
+                createTestPerson(4, Les_c4.EmployedOrSelfEmployed, Les_c4.EmployedOrSelfEmployed),
+                createTestPerson(5, Les_c4.EmployedOrSelfEmployed, Les_c4.NotEmployed),
+                createTestPerson(6, Les_c4.EmployedOrSelfEmployed, Les_c4.EmployedOrSelfEmployed),
+                createTestPerson(7, Les_c4.EmployedOrSelfEmployed, Les_c4.EmployedOrSelfEmployed),
+                createTestPerson(8, Les_c4.EmployedOrSelfEmployed, Les_c4.Retired),
+                // 50% from unemployment into employment
+                createTestPerson(9, Les_c4.NotEmployed, Les_c4.EmployedOrSelfEmployed),
+                createTestPerson(10, Les_c4.NotEmployed, Les_c4.EmployedOrSelfEmployed),
+                createTestPerson(11, Les_c4.NotEmployed, Les_c4.NotEmployed),
+                createTestPerson(12, Les_c4.NotEmployed, Les_c4.Retired),
+                // Ignore all rest as should be filtered out
+                createTestPerson(13, Les_c4.Student, Les_c4.Student),
+                createTestPerson(14, Les_c4.Retired, Les_c4.EmployedOrSelfEmployed),
+                createTestPerson(15, Les_c4.Retired, Les_c4.NotEmployed),
+                createTestPerson(16, Les_c4.Student, Les_c4.EmployedOrSelfEmployed)
+        );
+    }
+
+    @Test
+    @DisplayName("Proportion becoming unemployed")
+    public void proportionEmpToNotEmp() {
+
+
+        // Entering unemployment prevalence
+        EmploymentHistoryFilter employmentHistoryEmployed = new EmploymentHistoryFilter(Les_c4.EmployedOrSelfEmployed);
+        CrossSection.Integer personsEmpToNotEmp = new CrossSection.Integer(testPopulation, Person.class, "getNonwork", true);
+        personsEmpToNotEmp.setFilter(employmentHistoryEmployed);
+
+
+        MeanArrayFunction isEmpToNotEmp = new MeanArrayFunction(personsEmpToNotEmp);
+        isEmpToNotEmp.applyFunction();
+        assertEquals(0.25, isEmpToNotEmp.getDoubleValue(IDoubleSource.Variables.Default));
+
+    }
+
+    @Test
+    @DisplayName("Proportion becoming employed")
+    public void proportionNotEmpToEmp() {
+
+        // Entering employment prevalence
+        EmploymentHistoryFilter employmentHistoryUnemployed = new EmploymentHistoryFilter(Les_c4.NotEmployed);
+        CrossSection.Integer personsNotEmpToEmp = new CrossSection.Integer(testPopulation, Person.class, "getEmployed", true);
+        personsNotEmpToEmp.setFilter(employmentHistoryUnemployed);
+
+        MeanArrayFunction isNotEmpToEmp = new MeanArrayFunction(personsNotEmpToEmp);
+        isNotEmpToEmp.applyFunction();
+        assertEquals(0.5, isNotEmpToEmp.getDoubleValue(IDoubleSource.Variables.Default));
+
+
+    }
```

Comment:
I may be missing something, but am I right in saying these tests aren't actually using the `EmploymentStatistics` class? They're just duplicating the logic inside it. If so, changing that class wouldn't be able to break these tests, so I'm not sure they're really doing what they claim to do?

Reply:
Good point. Calculating statistics right now needs a `model` object passed to the `EmploymentStatistics.update()` model, so also a bit stuck till we can mock that.

Inclined to leave these parts in as placeholders, but can add a comment that these tests aren't sufficient and should be replaced when mock models are available?

Reply:
yeah want to try it as part of this?

Reply:
If useful, this is why I created a test BenefitUnit class (to mimic the regression model Person one) and managed to get it working in the UC mental health branch:

https://github.com/centreformicrosimulation/SimPaths/blob/ef8f743b20ac5b6f79888b84ceefdc6af1fdd144/src/test/java/simpaths/model/PersonTest.java#L202-L209

This was the minimal setup I needed to get a linked Person/BenefitUnit/Household to run a test, ignoring most of the baggage of all three